### ISSUE TITLE
v0.3.0 produces other results than v0.2.2

### ISSUE THREAD

**Issue by hendriks73** [bmcfee/resampy#101]
We just stumbled over this by accident and it's (probably) completely inconsequential, but v0.2.2 produces slightly different results from v0.3.0.

To reproduce, unzip the archive and run:

```
conda env create -f resampy022.yml; conda env create -f resampy030.yml; conda activate resampy022; python resampy_test.py; conda activate resampy030; python resampy_test.py;
```

Perhaps this is an intentional side-effect of some other change.

[Archive.zip](https://github.com/bmcfee/resampy/files/9045425/Archive.zip)

**Comment by bmcfee** [comment]
> We just stumbled over this by accident and it's (probably) completely inconsequential, but v0.2.2 produces slightly different results from v0.3.0.
> 
> To reproduce, unzip the archive and run:
> 
> ```
> conda env create -f resampy022.yml; conda env create -f resampy030.yml; conda activate resampy022; python resampy_test.py; conda activate resampy030; python resampy_test.py;
> ```
> 
> Perhaps this is an intentional side-effect of some other change.
> 
> [Archive.zip](https://github.com/bmcfee/resampy/files/9045425/Archive.zip)
> 
> 

Yes, this is known and expected. Per the changelog, the filters have been revised in the 0.3 release to reduce aliasing.

**Comment by bmcfee** [comment]
No worries!  Documentation probably could have been clearer here.
